modules/gapi/include/opencv2/gapi/fluid/gfluidbuffer.hpp

```diff
@@ -53,6 +53,8 @@ class GAPI_EXPORTS View
 
         inline const uint8_t* linePtr(int index) const
         {
+            GAPI_DbgAssert(index >= -m_border_size
+                        && index <  -m_border_size + static_cast<int>(m_linePtrs.size()));
```

Comment:
Please document this assert. Not clear to me how index and border size are related each other.

Reply:
Done